From c35a3387ebeff116bbaa296104f4f41782f9ea3a Mon Sep 17 00:00:00 2001
From: Marian Chereji <marian.chereji@freescale.com>
Date: Tue, 4 Dec 2012 15:59:58 +0200
Subject: [PATCH 432/518] dpa_classifier_wrapper: Optimize function return
 statements

Replace useless

ret = <return_code>;
return ret;

with

return <return_code>;

Signed-off-by: Marian Chereji <marian.chereji@freescale.com>
[Grabbed from the branch, LINUX_IR5.2.0, of
https://git.freescale.com/git-private/cgit.cgi/ppc/alu-b4860/linux.git.]
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 .../staging/fsl_dpa_offload/wrp_dpa_classifier.c   |    9 +++------
 1 files changed, 3 insertions(+), 6 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/wrp_dpa_classifier.c b/drivers/staging/fsl_dpa_offload/wrp_dpa_classifier.c
index 3987aa7..ea11ff9 100644
--- a/drivers/staging/fsl_dpa_offload/wrp_dpa_classifier.c
+++ b/drivers/staging/fsl_dpa_offload/wrp_dpa_classifier.c
@@ -1572,8 +1572,7 @@ static long do_ioctl_mcast_create_group(unsigned long args, bool compat_mode)
 #else
 	pr_err("ERROR: %s, %s (%d): Multicast not supported  on this"
 	       "platform.\n", __FILE__, __func__, __LINE__);
-	ret = -EINVAL;
-	return ret;
+	return -EINVAL;
 #endif
 
 #ifdef CONFIG_COMPAT
@@ -1627,8 +1626,7 @@ static long do_ioctl_mcast_import_group(unsigned long args, bool compat_mode)
 #else
 	pr_err("ERROR: %s, %s (%d): Multicast not supported  on this"
 	       "platform.\n", __FILE__, __func__, __LINE__);
-	ret = -EINVAL;
-	return ret;
+	return -EINVAL;
 #endif
 
 #ifdef CONFIG_COMPAT
@@ -1703,8 +1701,7 @@ static long do_ioctl_mcast_add_member(unsigned long args, bool compat_mode)
 #else
 	pr_err("ERROR: %s, %s (%d): Multicast not supported  on this"
 	       "platform.\n", __FILE__, __func__, __LINE__);
-	ret = -EINVAL;
-	return ret;
+	return -EINVAL;
 #endif
 
 #ifdef CONFIG_COMPAT
-- 
1.7.5.4

